### ISSUE TITLE
cbflib does not compile on OS X El Capitan with up-to-date XCode

### ISSUE THREAD

**Issue by graeme-winter** [yayahjb/cbflib#2]
Not due to code itself but due to TIFFLIB:

..... snip ....

```
duplicate symbol _isspace in:
    CMakeFiles/tiff.dir/libtiff/tif_print.c.o
    CMakeFiles/tiff.dir/libtiff/tif_sprint.c.o
duplicate symbol ___sputc in:
    CMakeFiles/tiff.dir/libtiff/tif_aux.c.o
    CMakeFiles/tiff.dir/libtiff/tif_sprint.c.o
duplicate symbol _isalpha in:
    CMakeFiles/tiff.dir/libtiff/tif_print.c.o
    CMakeFiles/tiff.dir/libtiff/tif_sprint.c.o
duplicate symbol ___sputc in:
    CMakeFiles/tiff.dir/libtiff/tif_aux.c.o
    CMakeFiles/tiff.dir/libtiff/tif_strip.c.o
duplicate symbol ___sputc in:
    CMakeFiles/tiff.dir/libtiff/tif_aux.c.o
    CMakeFiles/tiff.dir/libtiff/tif_swab.c.o
duplicate symbol ___sputc in:
    CMakeFiles/tiff.dir/libtiff/tif_aux.c.o
    CMakeFiles/tiff.dir/libtiff/tif_thunder.c.o
duplicate symbol ___sputc in:
    CMakeFiles/tiff.dir/libtiff/tif_aux.c.o
    CMakeFiles/tiff.dir/libtiff/tif_tile.c.o
duplicate symbol ___sputc in:
    CMakeFiles/tiff.dir/libtiff/tif_aux.c.o
    CMakeFiles/tiff.dir/libtiff/tif_version.c.o
duplicate symbol ___sputc in:
    CMakeFiles/tiff.dir/libtiff/tif_aux.c.o
    CMakeFiles/tiff.dir/libtiff/tif_warning.c.o
duplicate symbol ___sputc in:
    CMakeFiles/tiff.dir/libtiff/tif_aux.c.o
    CMakeFiles/tiff.dir/libtiff/tif_write.c.o
duplicate symbol ___sputc in:
    CMakeFiles/tiff.dir/libtiff/tif_aux.c.o
    CMakeFiles/tiff.dir/libtiff/tif_zip.c.o
duplicate symbol ___sputc in:
    CMakeFiles/tiff.dir/libtiff/tif_aux.c.o
    CMakeFiles/tiff.dir/libtiff/tif_unix.c.o
ld: 118 duplicate symbols for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[5]: *** [libtiff.dylib] Error 1
make[4]: *** [CMakeFiles/tiff.dir/all] Error 2
make[3]: *** [all] Error 2
make[2]: *** [tiff-4.0.3-rev-29Sep13-prefix/src/tiff-4.0.3-rev-29Sep13-stamp/tiff-4.0.3-rev-29Sep13-build] Error 2
make[1]: *** [CMakeFiles/tiff-4.0.3-rev-29Sep13.dir/all] Error 2
make: *** [all] Error 2
```

This is a known problem with TIFFLIB I think

Configuration:

```
CBF_USE_FORTRAN=NO /Applications/CMake.app/Contents/bin/cmake ../cbflib/
make
```

testing later tifflib now (seems to be up to 4.0.6 on ftp://download.osgeo.org/libtiff - canonical source today?)

**Comment by graeme-winter** [comment]
OK, that version seems to build correctly in isolation, old version fails with errors above; please could we consider updating to latest version which builds correctly?

**Comment by yayahjb** [comment]
Thanks -- I'll create an update of the modified tifflib that CBFlib uses.

Regards,
  Herbert

**Comment by yayahjb** [comment]
Updated tifflib to patched version of 4.0.6 -- HJB
